### DISCUSSION TITLE
Song Of Storms - Piano Notes (Collaboration w/Multi)

### DISCUSSION THREAD

**Discussion by briaryoung** [Monika-After-Story/MonikaModDev#7795] [general - piano notes]
Multi wanted to make piano notes for the Song of Storms, and then I passed it over to him. Now after you play it, it rains! 

**DON'T FORGET!** 
- Convert the first .txt into a .json file.
- Convert the second .txt file into an .rpy file, then put the .rpy file in the `game` folder of your DDLC directory! <3

The .json: 
[Song of Storms.txt](https://github.com/Monika-After-Story/MonikaModDev/files/4411787/Song.of.Storms.txt)

The .rpy:
[song of storms.txt](https://github.com/Monika-After-Story/MonikaModDev/files/4411789/song.of.storms.txt)

Created by @multimokia and @kkrosie123 

Find more piano notes or suggest more [here!](https://github.com/Monika-After-Story/MonikaModDev/issues/4483) \ (>w  <   ) /

**The Guide: (COMING SOON)**

**Comment by stupidwolf13** [comment]
I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


File "game/Song.of.Storms.rpy", line 1: expected statement.
    {
        "name":"Song of Storms",
        "win_label": "mas_song_sos_start_storm",
        "fc_label": "mas_song_sos_start_storm",
        "verse_list": [0],
        "pnm_list": [
            {
                "text": "",
                "style": "monika_credits_text",
                "notes": [
                    "F4",
                    "A4",
                    "D5",
                    "F4",
                    "A4",
                    "D5"
                ],
                "express": "1dua",
                "postexpress": "1eua",
                "vis_timeout": 2.0,
                "verse": 0
            },
            {
                "text": "",
                "style": "monika_credits_text",
                "notes": [
                    "E5",
                    "F5",
                    "E5",
                    "F5",
                    "E5",
                    "C5",
                    "A4"
                ],
                "express": "1dua",
                "postexpress": "1eua",
                "vis_timeout": 2.0,
                "verse": 0
            }
        ]
    }

Ren'Py Version: Ren'Py 6.99.12.4.2187

**Comment by Omniscients3** [comment]
Hey, can you please post the notes for the song?

**Comment by PKMN-Hobby** [comment]
I know I'm a bit late to the party, but the notes to this song are as follows:
Q E Y Q E Y

U I U I U T E 

and assuming you have the rpy file installed as well, it will, indeed start to rain outside the room.
